packages/core/src/renderer/components/workloads-pods/columns/pods-cpu-usage-column.injectable.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Freelens Authors. All rights reserved.
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { podListLayoutColumnInjectionToken } from "@freelensapp/list-layout";
+import { getInjectable } from "@ogre-tools/injectable";
+import React from "react";
+import podStoreInjectable from "../../workloads-pods/store.injectable";
+import { COLUMN_PRIORITY } from "./column-priority";
+
+const columnId = "cpuUsage";
+
+export const podsUsedCpuColumnInjectable = getInjectable({
+  id: "pods-cpu-usage-column",
+  instantiate: (di) => ({
+    id: columnId,
+    kind: "Pod",
+    apiVersion: "v1",
+    priority: COLUMN_PRIORITY.CPU_USAGE,
+    content: (pod) => {
+      const podStore = di.inject(podStoreInjectable);
+      const metrics = podStore.getPodKubeMetrics(pod);
+      const cpuUsage = metrics.cpu
+      
+      return <span>{cpuUsage === 0 ? "N/A" : cpuUsage.toFixed(3)}</span>;
```

Comment:
I'm wondering whether it's appropriate to show N/A when the CPU value is 0, given that 0 is the default value.
Wouldn't it make more sense to show N/A only when the metric server is unavailable or the data can't be retrieved?

For reference, memory currently displays N/A when the value is 0, so this change was made for consistency.